justfile

```diff
@@ -45,6 +57,12 @@ deps: dart-deps rust-deps install-async-bindgen
 dart-fmt:
     cd "$DART_WORKSPACE"; \
     dart format {{ if env_var_or_default("CI", "false") == "true" { "--output=none --set-exit-if-changed" } else { "" } }} .
+    cd "$FLUTTER_WORKSPACE"; \
+    dart format {{ if env_var_or_default("CI", "false") == "true" { "--output=none --set-exit-if-changed" } else { "" } }} .
```

Comment:
Can we put
`{{ if env_var_or_default("CI", "false") == "true" { "--output=none --set-exit-if-changed" } else { "" } }}` in a variable and pass that to `dart format`?

Reply:
We could, but I felt like it's not worth it for just one copy of the thing. If you think this is a must-have I can implement it though.

Reply:
not must-have